homeassistant/components/islamic_prayer_times/const.py

```diff
@@ -1,28 +1,38 @@
 """Constants for the Islamic Prayer component."""
 from typing import Final
 
-from prayer_times_calculator import PrayerTimesCalculator
-
 DOMAIN: Final = "islamic_prayer_times"
 NAME: Final = "Islamic Prayer Times"
 
 CONF_CALC_METHOD: Final = "calculation_method"
-CALC_METHODS: list[str] = list(PrayerTimesCalculator.CALCULATION_METHODS)
+CALC_METHODS: list[str] = [
+    "jafari",
+    "karachi",
+    "isna",
+    "mwl",
+    "makkah",
+    "egypt",
+    "tehran",
+    "gulf",
+    "kuwait",
+    "qatar",
+    "singapore",
+    "france",
+    "turkey",
+    "russia",
+    "moonsighting",
+    "custom",
+]
 DEFAULT_CALC_METHOD: Final = "isna"
 
 CONF_LAT_ADJ_METHOD: Final = "latitude_adjustment_method"
-LAT_ADJ_METHODS: list[str] = [
-    # PrayerTimesCalculator.LAT_ADJ_METHODS has values with space in them,
-    # which goes against translation_key requirement
-    value.replace(" ", "_")
-    for value in list(PrayerTimesCalculator.LAT_ADJ_METHODS)
-]
-DEFAULT_LAT_ADJ_METHOD: Final = "middle_of_the_night"
+LAT_ADJ_METHODS: list[str] = ["middle of the night", "one seventh", "angle based"]
+DEFAULT_LAT_ADJ_METHOD: Final = "middle of the night"
```

Comment:
These two lines need to replace space with underscore.

Reply:
@alyyousuf7 I just pushed update to fix this, sorry for the delay